Gemfile

```diff
@@ -13,7 +13,7 @@ group :development do
   gem 'komenda', '~> 0.1.6'
   gem 'launchy'
   gem 'rake', '10.4.2'
-  gem 'rubocop'
+  gem 'rubocop', '~> 0.43.0'
```

Comment:
I don't know about the environment, but it seems version `0.42.0` not working for me at least please check your version.

More details here https://github.com/projectatomic/vagrant-service-manager/pull/391#issuecomment-250703761